homeassistant/components/sensor/neurio_energy.py

```diff
@@ -21,48 +24,133 @@
 CONF_API_SECRET = 'api_secret'
 CONF_SENSOR_ID = 'sensor_id'
 
-DEFAULT_NAME = 'Energy Usage'
+ACTIVE_NAME = 'Energy Usage'
+DAILY_NAME = 'Daily Usage'
```

Comment:
Wondering if Daily Energy Usage and Daily Energy Usage would work better for the default names.